surveyor/surveyor.go

```diff
@@ -451,6 +452,14 @@ func (s *Surveyor) startObservations() error {
 			return fmt.Errorf("could not create observation from %s: %s", path, err)
 		}
 
+		// Prevent an equal observation to be loaded twice
+		// This is a problem that occurs with k8s mounts
+		for _, existingObservation := range s.observations {
+			if cmp.Equal(obs.opts, existingObservation.opts) {
```

Comment:
maybe just dedup on name rather than a full compare, anyway we should only allow a single one with each name so thats probably fine?

Reply:
That's a good suggestion. Wasn't sure if observations with the same should be possible!